server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -192,6 +192,7 @@ public class PostgresWireProtocol {
 
     public static int SERVER_VERSION_NUM = 100500;
     public static String PG_SERVER_VERSION = "10.5";
+    public static String DATE_STYLE = "ISO";
```

Comment:
Is this needed, since it's added to `Constants`?

Reply:
no, not needed anymore